### PR TITLE
Remove Emacs modes

### PR DESCRIPTION
**Pull request by Zimmi48**
**Kind:** clean-up

- [x] Corresponding documentation was added / updated (including any warning and error messages added / removed / modified).
- [x] Entry added in CHANGES.

On top of #7842 to avoid conflicts.

### CODE REVIEW

**Comment by ejgallego** [comment]
Just let me note that some time ago I think people were using to integrate Coq with `org-mode`, maybe it was @Matafou ? Just in case.

**Comment by Matafou** [comment]
Not me, it was Alan Shmitt. I don't know if he is still interested in this.

**Comment by gares** [comment]
> Not me, it was Alan Shmitt. I don't know if he is still interested in this.

I think @Zimmi48 should mention this in CHANGES.

**Comment by Zimmi48** [comment]
Good point. Added.

**Comment by ejgallego** [comment]
Maybe we should ping Alan? He could be interested in maintaining it.

**Review by gares — APPROVED** [review]

**Comment by gares** [comment]
Even if the file is maintained, I don't see why it should be in Coq proper. PG is not in Coq, for example.

**Comment by Matafou** [comment]
Maybe this should go in melpa as a coq-light-mode package?

**Comment by Zimmi48** [comment]
If there are users, we can help us set an external repository to maintain this (for instance as part of https://github.com/coq-community) and publish a MELPA package from there. But it doesn't really make sense to keep it part of Coq source code.
If you want, I can extend the CHANGES notes to say exactly what I've just said.

**Comment by Matafou** [comment]
Fair enough. If it happens that this thing has users we may decide to make Coq-light-mode a sub-distribution of pg itself. With a good hope that it will sync better with Coq.

**Comment by Zimmi48** [comment]
So should I add a note?

**Comment by Matafou** [comment]
Yes you should IMHO.

**Comment by Zimmi48** [comment]
Done and rebased.

**Review by ejgallego — APPROVED** [review]
Configure changes are fine.

**Comment by Zimmi48** [comment]
Ready to go. Who's self-assigning?

**Review by maximedenes — APPROVED** [review]

File: CHANGES [hunk 1]
```diff
@@ -45,6 +45,13 @@ Tools
   COQFLAGS is now entirely separate from COQLIBS, so in custom Makefiles
   $(COQFLAGS) should be replaced by $(COQFLAGS) $(COQLIBS).
 
+- The Emacs modes distributed with Coq where removed. You are advised to
```
Comment (maximedenes) [inline CHANGES]: where -> were

**Review by maximedenes — CHANGES_REQUESTED** [review]
typo to be fixed and quick rebase needed

**Comment by Zimmi48** [comment]
Fixed and rebased.

**Review by maximedenes — APPROVED** [review]

**Pull request merged**